### ISSUE TITLE
Getting 'Arguments to path.resolve must be strings' for every command

### ISSUE THREAD

**Issue by TheLudd** [mgan59/mongoose-fixture#4]
I installed this using

```
sudo npm install -g mongoose-fixture
```

Then I ran

```
mongoose-fixture --generateConfig
```

And I get a stack trace with an error saying:

> path.js:313
>        throw new TypeError('Arguments to path.resolve must be strings');
> 
> TypeError: Arguments to path.resolve must be strings
>    at Object.exports.resolve (path.js:313:15)
>    at Object.realpathSync (fs.js:1207:18)
>    at run (/usr/local/lib/node_modules/mongoose-fixture/lib/MongooseFixtureCommand.js:78:34)
>    at Object.<anonymous> (/usr/local/lib/node_modules/mongoose-fixture/bin/mongoose-fixture:7:41)
>    at Module._compile (module.js:456:26)
>    at Object.Module._extensions..js (module.js:474:10)
>    at Module.load (module.js:356:32)
>    at Function.Module._load (module.js:312:12)
>    at Function.Module.runMain (module.js:497:10)
>    at startup (node.js:119:16)

I get the same error when running other commands as well. Version is 0.1.3.

**Comment by mgan59** [comment]
Lud,

  I'll investigate the code and see if there are glaring issued.  Ideally I'll try and get some tests in place, specifically around the boilerplate generators.  

  Could you maybe tell me what system you are on (linux,mac,windows)?  I'm not sure if the code will work on windows because of the file-system pathing.  Also, what version of node you are running?

Thanks

**Comment by TheLudd** [comment]
I am using ubuntu 13.04 and node 0.10.12.

**Comment by mgan59** [comment]
K, able to duplicate the issue on my node 0.10.12. Working on a fix.

**Comment by mgan59** [comment]
[Issue Fixed](https://github.com/mgan59/mongoose-fixture/commit/7a0df4e05f0421c2292e4a61e93f151aa7e5e896)  

Fix was included in pre-existing 0.2.0 work.  Plan to have all this finalized and published to npm this evening.  Will close the issue once everything is posted to npm.

**Comment by mgan59** [comment]
I published 0.2.0 to npm this morning which contains this fix and a test.

Thanks for filing the issue Ludwig.
